cmake/FindUnibilium.cmake

```diff
@@ -23,6 +23,10 @@ find_path(UNIBILIUM_INCLUDE_DIR unibilium.h
           PATHS ${PC_UNIBILIUM_INCLUDEDIR} ${PC_UNIBILIUM_INCLUDE_DIRS}
           ${LIMIT_SEARCH})
 
+if(NVIM_BUILD_STATIC OR WIN32)
```

Comment:
Why is "OR WIN32" still here?  Should be controlled by NVIM_BUILD_STATIC.

Reply:
Because Mingw64 only have static UNIBILIUM libs, never dynamics.

Reply:
![image](https://user-images.githubusercontent.com/542599/35516861-d2d61b46-0547-11e8-8c15-d44233e7490e.png)